addOns/spider/src/main/java/org/zaproxy/addon/spider/parser/SpiderHttpHeaderParser.java

```diff
@@ -51,9 +53,17 @@ public boolean parseResource(ParseContext ctx) {
         }
 
         // Link header - potentially multiple absolute or relative URLs in < >
-        // TODO replace when next core version available HttpHeader.LINK
-        String link = message.getResponseHeader().getHeader("Link");
-        if (link != null && !link.isEmpty()) {
+        List<HttpHeaderField> links =
+                message.getResponseHeader().getHeaders().stream()
```

Comment:
Is this based on data in the wild? I thought this header would appear just once (does no harm to try process them all anyway, but same could be said from the others).